src/status_im/extensions/capacities/components.cljs

```diff
@@ -25,26 +28,44 @@
     (js/clearTimeout id))
   (reset! current (js/setTimeout #(on-input-change-text on-change value) delay)))
 
-(defn input [{:keys [keyboard-type style error on-change change-delay placeholder placeholder-text-color selection-color
+(defn- in-range
+  ;; Determines whether a number is in a range
+  [n, start, end]
+  (cond
+    (nil? n) true
+    (and (nil? start) (nil? end)) true
+    (= n "") true
```

Comment:
Can be combined with nil check into string/empty?

Reply:
changed 👍

Reply:
it could be simplest form
`(or (string/blank? n) (and (nil? start) (nil? end)) (and (< n end) (> n start)) )`